ad-joining/README.md

```diff
@@ -26,9 +26,12 @@ The `register-computer` app obtains its configuration from the environment and s
 * `AD_DOMAINCONTROLLER` (optional): Name or IP of domain controller to use. If not specified, the app will locate a domain controller via DNS automatically. This variable should only be used for testing.
 * `AD_USERNAME` (required): Username of Active Directory service user. The user must have permission to manage computer objects in the OUs used for automatic domain joining. The value must be provided in `NETBIOS-DOMAIN\SAM-ACCOUNT-NAME` format, UPNs are not supported.
 * `AD_PASSWORD` (optional): Clear-text password of Active Directory service user.  This variable should only be used for testing.
-* `SECRET_PROJECT_ID` (required unless `AD_PASSWORD` is used): Project ID under which the AD credentials have been stored in Secret Manager.
-* `SECRET_NAME` (required unless `AD_PASSWORD` is used): Name of the secret under which the AD credentials have been stored in Secret Manager.
-* `SECRET_VERSION` (required unless `AD_PASSWORD` is used): Version of the secret under which the AD credentials have been stored in Secret Manager. Can be set to `latest`.
+* `USE_LDAPS`: If set to `true` TLS secured LDAP on port 689 will be used when communicating with Active Directory. Defaults to `false`. *Note:* When using LDAPS and a private Certification Authority make sure the public CA certificate is uploaded to Secret Manager and the `SM_NAME_CACERT` and `SM_VERSION_CACERT` options are set accordingly.
+* `SM_PROJECT` (or `SECRET_PROJECT_ID` for backward compatibility): Project ID under which secrets are stored.
```

Comment:
I think we should demand that the secret is in the same project as the Cloud Run app.

I think Cloud Run sets `GOOGLE_CLOUD_PROJECT` to the current project -- if not, we'd have to look it up from the metadata server.

Reply:
This would break the existing behavior and I can see scenarios where Secret Manager might be in a different project than Cloud Run. I'm deferring this for the moment but we can revisit this in the future.